direct-hs/src/Web/Direct/Types.hs

```diff
@@ -6,19 +6,21 @@ import           Data.Word (Word64)
 ----------------------------------------------------------------
 
 -- | Domain ID.
-type DomainId  = Word64
```

Comment:
Please don't mix up functional changes and style changes. They should be separated into multiple commits.